src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,20 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        boolean isNegative = value < 0;
+        if (isNegative) {
+            value = -value;
+        }
+        StringBuilder result = new StringBuilder();
+        for (int i = value; i > 0; i /= 2) {
+            result.append(i % 2);
+        }
+        if (isNegative) {
+            result.append("-");
```

Comment:
negative numbers have the other form in the binary system (without minus) 
this task is waiting for only positive number, so let's remove implementation for negative number

Reply:
done